test/templates/input-group/index.njk

```diff
@@ -148,7 +148,7 @@
                 value: "1000000"
             }
         },
-        after: '<span class="tn-icon tn-icon--foo tn-has-background-color-text" style="width: 26px; height: 26px;" aria-label==""></span>'
+        after: '<span class="tn-icon tn-icon--foo tn-has-background-color-text" style="width: 26px; height: 26px;" aria-label=""></span>'
```

Comment:
This change isn't needed now, this has been replaced in #933 with an actual icon. Just make sure that version is used when fixing conflicts.